govc/tags/association/ls.go

```diff
@@ -30,6 +30,8 @@ import (
 
 type ls struct {
 	*flags.DatacenterFlag
+	name bool
```

Comment:
Let's remove the `name` flag and list names by default, a flag should not be required for an `ls` command. Just like the existing `govc ls` and `govc find` commands, they list name by default and the `-i` flag switches to listing the ID.

Reply:
Updated

Reply:
Take ```$ govc tags.ls``` for example:
```$ govc tags.ls``` will return only tags names
```$ govc tags.ls -i``` will return only tags IDs
```$ govc tags.ls -json | jq .``` will return something like this :
```
 {
   TagID: "urn:...", 
   Name: "CA-K8S"
 },
 {
    TagID: "urn:...", 
    Name: "WA-K8S"
 }
]
```
```$ govc tags.ls -c CATEGORYID``` will return only tags names in this category
```$ govc tags.ls -c CATEGORYID -json | jq .```will return the JSON like above

Reply:
ok, then we can remove the `ls.name` field here, right?